src/lib/cache/subCache.cpp

```diff
@@ -720,6 +720,16 @@ void subCacheItemInsert
   cSubP->expression.coords     = coords;
   cSubP->expression.georel     = georel;
 
+  std::string errorString;
+  if (q != "")
+  {
+    if (cSubP->expression.stringFilter.parse(q.c_str(), &errorString) == false)
+    {
+      LM_E(("Runtime Error (error parsing q-string '%s': %s)", q.c_str(), errorString.c_str()));
```

Comment:
StringFilter::parse takes care of all error handling.
And, true, it is not a 'Runtime Error' but a 'Bad Input' error.